src/wings_geom_win.erl

```diff
@@ -692,14 +697,18 @@ unsplit_window(Splitter, TC) ->
 update_folders({Curr, Fld0}, TC) ->
     Do = fun() ->
 		 wxTreeCtrl:deleteAllItems(TC),
-		 [{no_folder,_,S0}|Fld] = Fld0,
+		 [{no_folder,_,S0,_}|Fld] = Fld0,
 		 Caption0 = io_lib:format("~ts (~p)",[?__(1, "Objects"), S0]),
 		 Root = wxTreeCtrl:addRoot(TC, Caption0, []),
-
-		 Sorted = lists:sort([{wings_util:cap(F),F,S} || {F,_,S} <- Fld]),
-		 Add = fun({_, Name, Qtd}) ->
+		 Sorted = lists:sort([{wings_util:cap(F),F,S,Hs} || {F,_,S,Hs} <- Fld]),
+		 Add = fun({_, Name, Qtd, HasSel}) ->
 			    Caption = io_lib:format("~ts (~p)",[Name, Qtd]),
-			    {wxTreeCtrl:appendItem(TC, Root, Caption, []), Name}
+                Options =
+                    case HasSel of
+                        true -> [{image, image_idx(sel)},{selectedImage, image_idx(sel)}];
+                        false -> []
+                    end,
```

Comment:
Indentation looks wrong here at github at least

Reply:
> Have you tested this anywhere else besides windows?
> 
> Cause this looks like things that will break on some OS.

No. I'm going to try some help with that.

Reply:
Yes

Reply:
> Indentation looks wrong here at github at least

I fixed the indentation in the piece I worked by copying from the previous line.
This file (as well as many others) has a mix of _tab_ and _space_ chars - easy to see when in the editor we enable the option to show them.

Reply:
Sorry about that, new edits should have space only, no tabs on edited lines,
I'll take a look and fix it if needs more fixes.